README_LLaMA_BYOCTritonTRTLLM.md

```diff
@@ -0,0 +1,148 @@
+| `max_tokens_in_paged_kv_cache` | The maximum size of the KV cache in number of tokens |
+| `max_attention_window_size` | When using techniques like sliding window attention, the maximum number of tokens that are attended to generate one token. Defaults to maximum sequence length |
+| `batch_scheduler_policy` | Set to `max_utilization` to greedily pack as many requests as possible in each current in-flight batching iteration. This maximizes the throughput but may result in overheads due to request pause/resume if KV cache limits are reached during execution. Set to `guaranteed_no_evict` to guarantee that a started request is never paused.|
+| `kv_cache_free_gpu_mem_fraction` | Set to a number between 0 and 1 to indicate the maximum fraction of GPU memory (after loading the model) that may be used for KV cache|
+| `max_num_sequences` | Maximum number of sequences that the in-flight batching scheme can maintain state for. Defaults to `max_batch_size` if `enable_trt_overlap` is `false` and to `2 * max_batch_size` if `enable_trt_overlap` is `true`, where `max_batch_size` is the TRT engine maximum batch size.
+| `enable_trt_overlap` | Set to `true` to partition available requests into 2 'microbatches' that can be run concurrently to hide exposed CPU runtime |
+| `exclude_input_in_output` | Set to `true` to only return completion tokens in a response. Set to `false` to return the prompt tokens concatenated with the generated tokens  |
+
+#### Run the following command to prepare tensorrt_llm model config.pbtxt
+
+```
+python3 tools/fill_template.py -i llama_ifb/tensorrt_llm/config.pbtxt triton_max_batch_size:64,decoupled_mode:False,max_beam_width:1,engine_dir:/tmp/llama/7B/trt_engines/fp16/1-gpu/,max_tokens_in_paged_kv_cache:2560,max_attention_window_size:2560,kv_cache_free_gpu_mem_fraction:0.5,exclude_input_in_output:True,enable_kv_cache_reuse:False,batching_strategy:inflight_batching,max_queue_delay_microseconds:600
+```
+
+*triton_model_repo/postprocessing/config.pbtxt*
+
+Mandatory ones are
+| Name | Description
+| :----------------------: | :-----------------------------: |
+| `triton_max_batch_size` | Here setting to 4 |
+| `tokenizer_dir` | The path to the tokenizer for the model. In this example, the path should be set to `/workspace/tensorrtllm_backend/tensorrt_llm/examples/gpt/gpt2`|
+| `tokenizer_type` | The type of the tokenizer for the model, `t5`, `auto` and `llama` are supported. In this example, the type should be set to `auto` |
+| `postprocessing_instance_count` | Here setting to 1 |
+
+#### Run the following command to prepare postprocessing config.pbtxt
+```
+python3 tools/fill_template.py -i llama_ifb/postprocessing/config.pbtxt tokenizer_dir:${HF_LLAMA_MODEL},tokenizer_type:llama,triton_max_batch_size:64,postprocessing_instance_count:1
+```
+
+*triton_model_repo/ensemble/config.pbtxt*
+
+| Name | Description
+| :----------------------: | :-----------------------------: |
+| `triton_max_batch_size` | Here setting to 4 |

```

Comment:
I see you are setting it to 64 in all the config.pbtxt, can you please update the doc to say "Here setting to 64"